test/dag-node-test.js

```diff
@@ -80,7 +82,7 @@ module.exports = (repo) => {
       })
 
       const node2 = new DAGNode(someData, l2)
-      expect(node2.Links).to.eql([l1[1], l1[0]])
+      expect(node2.Links).to.containSubset([l1[1], l1[0]])
```

Comment:
Doesn't marking the properties non-enumerable mean you should be able to use `.deep.equal` (or `.eql`) here?

Reply:
I believe problem with `.deep.equal` and `.eql` is that fails if values do have different prototype chains.

Reply:
Turns out the problem here was caused by `[Symbol.toStringTag]` getter added by `class-is` library.

https://github.com/moxystudio/js-class-is/blob/master/index.js#L19-L21

I'm assuming `eql` somewhere does `Object.prototype.toString.call(v)` but unless this is a blocker, I would prefer not to dig into this any further.

Reply:
Could you open an issue against `class-is` please?  It should not get in the way like this.

Reply:
https://github.com/moxystudio/js-class-is/issues/27